index.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
- [ ] As the previous reviewer mentioned it is a requirement to pass the linters check. You could try with the command `npx stylelint "**/*.{css,scss}"` to see the issues locally and you could try `npx stylelint "**/*.{css,scss}" --fix` to fix them (if possible) :smiley: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/143038155-6f7bc338-fd9a-43df-aeba-947639324942.png)